content/docs/examples/multicluster/icp/index.md

```diff
@@ -148,7 +148,7 @@ across all nodes in the two IBM Cloud Private Clusters.
 
 ## Install Istio for multicluster
 
-[Follow the VPN-based multicluster installation steps](/docs/setup/kubernetes/install/multicluster/vpn/) to install and configure
+[Follow the VPN-based multicluster installation steps](/docs/setup/kubernetes/install/multicluster/shared-vpn/) to install and configure
```

Comment:
Follow the "single-network shared control plane installation steps"?  See there is a bit of inconsistency.